conda/cli/conda_argparse.py

```diff
@@ -1779,19 +1797,28 @@ def add_parser_prune(p):
     )
 
 
-def add_parser_experimental_solver(p):
+def add_parser_solver(p):
     """
     Add a command-line flag for alternative solver backends.
 
-    See ``context.experimental_solver`` for more info.
+    See ``context.solver`` for more info.
 
     TODO: This will be replaced by a proper plugin mechanism in the future.
     """
-    p.add_argument(
+    group = p.add_mutually_exclusive_group()
+    group.add_argument(
+        "--solver",
+        dest="solver",
+        choices=[v.value for v in SolverChoice],
+        help="Choose which solver backend to use.",
+        default=NULL,
+    )
+    group.add_argument(
         "--experimental-solver",
-        dest="experimental_solver",
-        choices=[v.value for v in ExperimentalSolverChoice],
-        help="EXPERIMENTAL. Choose which solver backend to use.",
+        action=PendingDeprecationAction,
+        dest="solver",
+        choices=[v.value for v in SolverChoice],
+        help="DEPRECATED. Please use '--solver' instead.",
```

Comment:
Even though we have the `deprecated` message, should we still have what this is intended to do? "Choose which solver backend to use."

Reply:
They show up together in the help message, so I don't know if repeating it again helps clarify or adds noise:

```
Solver Mode Modifiers:
  --strict-channel-priority
                        Packages in lower priority channels are not considered if a package with the same name appears in a higher priority channel.
  --no-channel-priority
                        Package version takes precedence over channel priority. Overrides the value given by `conda config --show channel_priority`.
  --no-deps             Do not install, update, remove, or change dependencies. This WILL lead to broken environments and inconsistent behavior. Use at your own risk.
  --only-deps           Only install dependencies.
  --no-pin              Ignore pinned file.
  --solver {classic,libmamba,libmamba-draft}
                        Choose which solver backend to use.
  --experimental-solver {classic,libmamba,libmamba-draft}
                        DEPRECATED. Please use '--solver' instead.
  --force-reinstall     Ensure that any user-requested package for the current operation is uninstalled and reinstalled, even if that package already exists in the environment.
  --freeze-installed, --no-update-deps
                        Do not update or change already-installed dependencies.
  --update-deps         Update dependencies that have available updates.
  -S, --satisfied-skip-solve
                        Exit early and do not run the solver if the requested specs are satisfied. Also skips aggressive updates as configured by the 'aggressive_update_packages' config setting. Use 'conda info --describe
                        aggressive_update_packages' to view your setting. --satisfied-skip-solve is similar to the default behavior of 'pip install'.
  --update-all, --all   Update all installed packages in the environment.
  --update-specs        Update based on provided specifications.
```